SYCL/InvokeSimd/feature/call_vadd_1d_noret.cpp

```diff
@@ -0,0 +1,165 @@
+// TODO: enable on Windows once driver is ready
+// REQUIRES: gpu && linux
+// UNSUPPORTED: cuda || hip
+//
+// Check that full compilation works:
+// RUN: %clangxx -fsycl -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr %s -o %t.out
+// RUN: env IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %GPU_RUN_PLACEHOLDER %t.out
+//
+// VISALTO enable run
+// RUN: env IGC_VISALTO=63 IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %GPU_RUN_PLACEHOLDER %t.out
+
+// Tests invoke_simd support in the compiler/headers
+/* Since this program tests "no return" functionality, ideally, this function
```

Comment:
call_vadd_1d_noret.cpp was created at the time of early days of InvokeSimd development to get around the failure of void_retval.cpp. I believe it can be safely removed now.